suites/upgrade/infernalis/older/2-workload/blogbench.yaml

```diff
@@ -0,0 +1,11 @@
+meta:
+   - desc: |
+      run a filesystem stress test
+      (not running on cephfs)
```

Comment:
should update the description to say 'run a cephfs stress test' now that it has ceph-fuse